ui/apps/platform/src/Containers/Administration/Events/AdministrationEventsToolbar.tsx

```diff
@@ -1,27 +1,53 @@
 import React, { ReactElement } from 'react';
 import {
     Pagination,
+    Text,
     Toolbar,
     ToolbarContent,
     ToolbarGroup,
     ToolbarItem,
 } from '@patternfly/react-core';
+import pluralize from 'pluralize';
 
 export type AdministrationEventsToolbarProps = {
-    count: string; // int64
+    count: number;
+    page: number;
+    perPage: number;
+    setPage: (newPage: number) => void;
+    setPerPage: (newPerPage: number) => void;
 };
 
-function AdministrationEventsToolbar({ count }: AdministrationEventsToolbarProps): ReactElement {
-    // TODO table filters and pagination
-    const page = 1;
-    const perPage = 10;
+function AdministrationEventsToolbar({
+    count,
+    page,
+    perPage,
+    setPage,
+    setPerPage,
+}: AdministrationEventsToolbarProps): ReactElement {
+    const countText = `${count} ${pluralize('event', count)} found`;
 
     return (
         <Toolbar>
             <ToolbarContent>
+                <ToolbarGroup>
+                    <ToolbarItem variant="label">
+                        <Text>{countText}</Text>
+                    </ToolbarItem>
+                </ToolbarGroup>
                 <ToolbarGroup alignment={{ default: 'alignRight' }}>
                     <ToolbarItem variant="pagination">
-                        <Pagination itemCount={Number(count) ?? 0} page={page} perPage={perPage} />
+                        <Pagination
+                            itemCount={count}
+                            page={page}
+                            perPage={perPage}
+                            onSetPage={(_, newPage) => setPage(newPage)}
+                            onPerPageSelect={(_, newPerPage) => {
+                                if (count < (page - 1) * newPerPage) {
+                                    setPage(1);
+                                }
```

Comment:
I've used this same pattern elsewhere, but I wonder for general UX if it would be better to _always_ set the page to `1` when the user changes the "per page" value.

Reply:
Indeed, that was copy paste. I agree and will change in the next step.

Reply:
Changing now to rerun CI and then merge.